### PR TITLE
fix: [PL-40071]: uicore version bump

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by avinashmadhwani02** [comment]
retrigger titlecheck

File: configs/devServerProxy.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-const baseUrl = process.env.BASE_URL ?? 'https://qa.harness.io/gateway'
+const baseUrl = process.env.BASE_URL ?? 'https://app.harness.io/gateway'
```
Comment (rameshx) [inline configs/devServerProxy.config.js:8]: Is this intentional?

Comment (knagurski) [reply configs/devServerProxy.config.js:8]: We don't want to default to trying to use a live production system.
```suggestion
const baseUrl = process.env.BASE_URL ?? 'https://qa.harness.io/gateway'
```

Comment (avinashmadhwani02) [reply configs/devServerProxy.config.js]: It was added by mistake. removed it now

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**